nilearn/plotting/img_plotting.py

```diff
@@ -312,6 +333,116 @@ def _load_anat(anat_img=MNI152TEMPLATE, dim=False, black_bg='auto'):
+        display_mode: {'ortho', 'x', 'y', 'z'}
+            Choose the direction of the cuts: 'x' - saggital, 'y' - coronal,
+            'z' - axial, 'ortho' - three cuts are performed in orthogonal
+            directions.
+        figure : integer or matplotlib figure, optional
+            Matplotlib figure used or its number. If None is given, a
+            new figure is created.
+        axes : matplotlib axes or 4 tuple of float: (xmin, ymin, width, height), optional
+            The axes, or the coordinates, in matplotlib figure space,
+            of the axes used to display the plot. If None, the complete
+            figure is used.
+        title : string, optional
+            The title displayed on the figure.
+        annotate: boolean, optional
+            If annotate is True, positions and left/right annotation
+            are added to the plot.
+        draw_cross: boolean, optional
+            If draw_cross is True, a cross is drawn on the plot to
+            indicate the cut plosition.
+        black_bg: boolean, optional
+            If True, the background of the image is set to be black. If
+            you wish to save figures with a black background, you
+            will need to pass "facecolor='k', edgecolor='k'" to pylab's
+            savefig.
+        cmap: matplotlib colormap, optional
+            The colormap for the atlas maps 
+        vmin: float
+            Lower bound for plotting, passed to matplotlib.pyplot.imshow
+        vmax: float
+            Upper bound for plotting, passed to matplotlib.pyplot.imshow
+
+    """
+    anat_img, black_bg, _, _ = _load_anat(anat_img, dim=dim, 
+                                          black_bg=black_bg)
+    display = plot_img(anat_img, cut_coords=cut_coords,
+                       output_file=output_file, display_mode=display_mode,
+                       figure=figure, axes=axes, title=title,
+                       threshold=None, annotate=annotate,
+                       draw_cross=draw_cross, black_bg=black_bg,
+                       cmap=plt.cm.gray, **kwargs) 
+ 
+    maps_img = _utils.check_niimg_4d(maps_img)
+    n_maps = maps_img.shape[3]
+   
+    if isinstance(cmap, basestring):
+       color_map = plt.cm.get_cmap(cmap)
+    # Build a custom colormap for displaying contours
+    color_list = color_map(np.linspace(0, 1, n_maps))
+
+    for i, (map_img, color) in enumerate(zip(iter_img(maps_img), color_list)):
+        over = black_to(color)
+        cont = ListedColormap(color).colors
+        if threshold is not None:
+           if hasattr(threshold, '__iter__'):
+              value = threshold[i]
+           else: 
+                value = threshold
+        # To threshold or choose the level of the contours
+        percentile = 100 * (1 - (value/n_maps))
+        data = np.abs(map_img.get_data())
+        level = fast_abs_percentile(data, percentile) + 1e-4
+        
+        if view_type == 'auto': 
```

Comment:
Yes. And contours should be filled contours, like what Yannick does. Filled contours are not continous overlays, they are obtained using the matplotlib contourf function.